### PR TITLE
RFC: Update release process for nteract

### PR DESCRIPTION
**Pull request by captainsafia**
After spending the fast few months interacting with our updated release cadence, interacting with users about release management, and more I've drafted this proposal for releasing on nteract.

cc: @wadethestealth @rgbkrk

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,18 +1,49 @@
...
-Unless there is an emergency bug or security fix, packages are released on a weekly cadence on Mondays. The desktop app is released on a bi-weekly cadence on Mondays. So, every Monday a package release is cut and every other Monday a desktop release is cut.
+This document outlines logistics around how releases are handled in the nteract project, specifically the applications and npm packages within this repository.
 
-## Releasing the Packages
-In order to release the JavaScript packages to npm, you will need to run the `npx lerna publish` command in the root directory. This will initialize the Lerna release process which will prompt you to set the new version numbers for each package.
+## Release Frequency
 
-## Releasing the Desktop App and Packages
+Production releases of nteract's npm packages, desktop app, and Jupyter extension will happen on a monthly cadence. Dev releases of nteract's npm packages will happen on a weekly basis. This segmentation allows users who are eager to grab the latest changes to do so at the existing pace of the weekly release cadence. Users who want more stable and less frequent releases can opt to consume from the production tag.
```
Comment (rgbkrk) [inline RELEASING.md:7]: 👍

File: RELEASING.md [hunk 2]
```diff
@@ -1,18 +1,49 @@
...
+2. Create a new branch with the `release/yyyymmdd` naming convention.
+3. Run the `npx lerna publish --dist-tag=prerelease` command in the root of the nteract repo.
+4. Follow the prompts to incremenet the versions of each package as appropriate.
+   - Note that lerna will provide an output that lists the newly published versions of each package. Copy this as it will be relevant in later steps.
+5. Create a new Markdown file in the `changelogs` directory that captures the changes in this release.
+   - The filename for this Markdown file should follow the `YYYY-MM-DD.md` naming convention.
+   - The contents of the file should be based on the `changelogs/changelog_template.md` template file.
+6. Commit the changes with the new markdown file.
+7. Open a PR against master from your `release/yyyymmdd` branch. In the description of the PR, include the versions of the published npm packages that the `lerna publish` command provides.
+8. Merge the PR into master.
```
Comment (rgbkrk) [inline RELEASING.md:23]: Ok, cool, I'm loving this dev release process and that we can opt into using prereleases. Thanks for getting this working well.

File: RELEASING.md [hunk 3]
```diff
@@ -70,4 +101,13 @@ babbly
...
+6.  Once you're ready click "Publish release". On macOS and Windows, the update will be automatically downloaded and installed on user's machines.
+
+## Release Process for the nteract Jupyter Extension
+
+To release the nteract Jupyter Extension, you will need to complete the following steps.
+
+1. Navigate to the `applications/jupyter-extension` directory within the nteract repo.
+2. Ensure that you are signed into PyPi on your machine and have permissions to publish the `nteract_on_jupyter` package.
+3. Run the `yarn build:python` command.
+4. Run the `yarn upload:pyp` command.
```
Comment (rgbkrk) [inline RELEASING.md]: Sweet

**Review by rgbkrk — APPROVED** [review]

**Comment by captainsafia** [comment]
I'm gonna go ahead and merge this now. We'll start off using this process next Monday!

**Pull request merged**